sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/Metrics.cs

```diff
@@ -199,6 +155,82 @@ internal class Metrics
     ///
     public const string EventsProcessed = "EventsProcessed";
 
+    /// <summary>
+    ///   This is the metric name used to collect metrics on the number of duplicate events processed by the processor.
+    /// </summary>
+    ///
+    public const string DuplicateEventsDiscarded = "DuplicateEventsDiscarded";
```

Comment:
Not important, but if you wanted to cheat and make sure that any updates to names are captured as they happen, you could do:

```csharp
public static readonly string DuplicateEventsDiscarded = nameof(DuplicateEventsDiscarded);
```

That said, I don't think it's worth doing here since we aren't likely to refactor often.   Just thought I'd mention it for future considerations.